docs/application/dotnet/guides/nui/text.md

```diff
@@ -0,0 +1,590 @@
+
+The `TextLabel` class is lightweight, non-editable, and do not respond to user input. Text labels support multiple languages and scripts including right-to-left scripts such as Arabic. For more information on how to display a text using a text label, see [NUI Hello World Tutorial](hello-world.md).
+
+**Figure: Text label positioned to top left**
+
+![Text label example, positioned to top left](media/TextLabelTopLeft.png)
+
+<a name="create"></a>
+### Create Text Label
+
+To create a text label:
+
+1.  Create an instance of the [Tizen.NUI.BaseComponents.TextLabel](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.TextLabel.html) class and define the label text as a parameter:
+
+    ```csharp
+    TextLabel label = new TextLabel("Hello World");
+    ```
+
+    You can also create the `Tizen.NUI.BaseComponents.TextLabel` class instance separately and define the label text by setting its `Text` property:
+
+    ```csharp
+    TextLabel label = new TextLabel();
+    label.Text = "Hello World";
+    ```
+
+    > [!CAUTION]
+    >
+    > To display the label properly, the `Text` property must be a UTF-8 string. Any `CR+LF` new line characters are replaced by `LF`.
+
+2.  Define the label position on-screen with the `ParentOrigin` property of the `Tizen.NUI.BaseComponents.TextLabel` class:
+
+    ```csharp
+    label.ParentOrigin = ParentOrigin.TopLeft;
+    ```
+
+3.  Add the text label control to a window:
+
+    ```csharp
+    Window window = Window.Instance;
+    window.Add(label);
+    ```
+
+    > [!CAUTION]
+    >
+    > The `TextLabel` object can be only added to a `Window` object or a `View` object on a `Window` object.
+
+<a name="font"></a>
+### Select Fonts
+
+You can request a specific font using the `FontFamily`, the `FontStyle`, and the `PointSize` properties of the [Tizen.NUI.BaseComponents.TextLabel](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.TextLabel.html) class:
+
+-   `FontFamily` is a string with the font family name, such as `FreeSerif`.
+-   `FontStyle` is a JSON-formatted string with the font style. The following list describes some possible keys and common values for them:
+    -   The `width` key defines the spacing between glyphs. Some commonly-used values include `condensed`, `semiCondensed`, `normal`, `semiExpanded`, and `expanded`.
+    -   The `weight` key defines the thickness or darkness of the glyphs. Some commonly-used values include `thin`, `light`, `normal`, `regular`, `medium`, and `bold`.
+    -   The `slant` key defines whether to use italics. Some commonly used values include `normal`, `roman`, `italic`, and `oblique`.
+
+        > [!NOTE]
+        >
+        > Usually `italic` is a separate font, while `oblique` is generated by applying a slant to the `normal` font.
+
+
+-   `PointSize` is a type float. To calculate the point size from the height in pixels, use the following formula, where `vertical_dpi` is the vertical resolution of a device in dots per inch:

```

Comment:
This seems to be somewhat ambiguous 
`PointSize` is a type float.
Suggest you to use the following for more clarity.: 
`PointSize` is set as float.